content/guides/contribute/design.mdx

```diff
@@ -41,14 +39,13 @@ There are different ways you can ask for and get feedback:
 
 ## Design checklist
 
-All Design patterns for Primer Library consideration should at minimum:
-
-- [ ] **Leverage Primer**: Leverages the foundational building blocks and tools of Primer. Reuses or extends existing patterns and primitives rather than creating overrides 
-    - [ ] Colors, spacing, sizes, and typographic styles from @primer/primitives
-    - [ ] All icons come from @primer/octicons
-    - [ ] Responsive
-- [ ] **Maintains high design quality:** Visually in harmony with GitHub UI and all visual elements are used are in their correct context
-- [ ] **Prioritizes code as a top concern:** Designed with code quality and component API top of mind
-- [ ] **Is inclusive:** Designs follow Primer accessibility guidelines and consider a global audience 
-- [ ] **Has documentation:** Design patterns are [documented](https://primer.style/design/guides/contribute/documentation) for easy adoption
-  
+All design patterns for Primer Library consideration should at minimum:
+
+- [ ] **Leverage Primer**: Leverages the foundational building blocks and tools of Primer. Reuses or extends existing patterns and primitives rather than creating overrides
+  - [ ] Colors, spacing, sizes, and typographic styles from [@primer/primitives](https://github.com/primer/primitives/)
+  - [ ] All icons come from [@primer/octicons](https://github.com/primer/octicons/)
+  - [ ] Considers the criteria in the [responsive design guidelines](https://primer.style/design/foundations/responsive)
+- [ ] **Maintain high design quality:** Visually in harmony with GitHub UI and all visual elements are used are in their correct context
+- [ ] **Prioritize code as a top concern:** Designed with code quality and component API top of mind
+- [ ] **Is inclusive:** Designs follow Primer [accessibility guidelines](https://primer.style/design/guides/accessibility/guidelines) and considers a global audience
```

Comment:
```suggestion
- [ ] **Be inclusive:** Designs follow Primer [accessibility guidelines](https://primer.style/design/guides/accessibility/guidelines) and considers a global audience
```

ugh. I know this is an aside from the original PR intent - I think I wrote this (or the bullet below) but grammatically this feels weird - should maybe be `all design patterns at a minimum should **be inclusive**' (?) the rest of the bullets sound write as a continuation of the top point